Pipfile

```diff
@@ -10,4 +10,4 @@ Sphinx = {version = "*", markers = "python_version != '3.3'"}
 sphinx-autobuild = {version = "*", markers = "python_version != '3.3'"}
 
 [packages]
-"e1839a8" = {path = ".", extras = ["tests", "aiohttp", "marshmallow", "twisted", "typing"], editable = true}
+uplink = {editable = true, extras = ["aiohttp", "marshmallow", "pydantic", "tests", "twisted", "typing"], path = "."}
```

Comment:
Regarding the Travis CI failures: could they be related to this change?

Reply:
I've just replaced the package name "uplink" by the old hash commit `e1839a8`. Don't know what happened (there are some issues in pipenv related with this) but it worked. Thought for a moment about pipenv version, but it's impossible since versions 3.7 and 3.8 also failed due the same reason.